webapp/views/wpt-results.js

```diff
@@ -288,14 +287,6 @@ class WPTResults extends AmendMetadataMixin(Pluralizer(WPTColors(WPTFlags(PathIn
           <h3>
             History <span>(Experimental)</span>
           </h3>
-          <test-results-chart
```

Comment:
Also remove `/webapp/components/test-results-chart.js`. This is the only place that uses `test-results-chart`